### PR TITLE
update news on start page

### PR DESCRIPTION
**Pull request by marcoow**
This replaces the outdated news on the start page with announcements about JSConf and EmberFest.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,16 +39,16 @@ <h2>Solid engineering<br>for ambitious projects</h2>
       <div class="row content-section">
         <div class="col-sm-6 content-teaser">
           <div>
-            <h4>EmberConf 2018</h4>
-            <p><a href="http://emberconf.com">EmberConf 2018</a> is set for March 13th &amp; 14th in Portland, OR with <strong>simplabs engineers <a href="https://twitter.com/jjordan_dev">@jjordan_dev</a> and <a href="https://twitter.com/tobiasbieniek">@tobiasbieniek</a> among the speakers</strong>.</p>
-            <a href="http://emberconf.com" class="custom-button">learn more</a>
+            <h4>JSConf EU 2018, Berlin</h4>
+            <p>We are happy to be part of JSConf EU again in 2018 as a community sponsor. Visit us at our booth if you're in Berlin on June 2nd and 3rd - we have <strong><a href="https://twitter.com/locks">@locks</a> form the Ember.js core team</strong> join us and will present some cutting edge <strong>demos with Ember.js and Glimmer.js</strong>!</p>
```
Comment (jayjayjpg) [inline index.html]: I think this should be `@locks</a> from the Ember.js` - we could also write 
`Ember.js core team member @locks will join us and present some cutting edge [...]`

File: index.html [hunk 2]
```diff
@@ -39,16 +39,16 @@ <h2>Solid engineering<br>for ambitious projects</h2>
...
+            <a href="https://2018.jsconf.eu" class="custom-button">learn more</a>
           </div>
         </div>
         <div class="col-sm-6 content-teaser">
           <div>
-            <h4>Bulgaria Web Summit 2018, April 14th</h4>
-            <p><a href="https://twitter.com/jjordan_dev">@jjordan_dev</a> will talk about <strong>testing against time in JavaScript applications</strong> at Bulgaria Web Summit in Sofia this April.</p>
-            <a href="https://bulgariawebsummit.com/" class="custom-button">learn more</a>
+            <h4>EmberFest 2018, Amsterdam</h4>
+            <p>As in 2017, <strong>simplabs is part of the organisational committee behind EmberFest</strong>. This year the conference is headed to Amsterdam in the Netherlands and our team is looking forward to <strong>meeting up with there with the european Ember community on October 11th and 12th</strong>!</p>
```
Comment (jayjayjpg) [inline index.html]: I think this should be 

meeting up with ~there~

File: index.html [hunk 3]
```diff
@@ -39,16 +39,16 @@ <h2>Solid engineering<br>for ambitious projects</h2>
...
-            <a href="http://emberconf.com" class="custom-button">learn more</a>
+            <h4>JSConf EU 2018, Berlin</h4>
+            <p>We are happy to be part of JSConf EU again in 2018 as a community sponsor. Visit us at our booth if you're in Berlin on June 2nd and 3rd - we have <strong><a href="https://twitter.com/locks">@locks</a> form the Ember.js core team</strong> join us and will present some cutting edge <strong>demos with Ember.js and Glimmer.js</strong>!</p>
+            <a href="https://2018.jsconf.eu" class="custom-button">learn more</a>
           </div>
         </div>
         <div class="col-sm-6 content-teaser">
           <div>
-            <h4>Bulgaria Web Summit 2018, April 14th</h4>
-            <p><a href="https://twitter.com/jjordan_dev">@jjordan_dev</a> will talk about <strong>testing against time in JavaScript applications</strong> at Bulgaria Web Summit in Sofia this April.</p>
```
Comment (jayjayjpg) [inline index.html:50]: If an adjective is derived from a [proper noun it should be capitalized](https://en.wikipedia.org/wiki/Proper_noun#Capitalization) and proper nouns also include country and region names.

> Capital letters are used:
> [...]
> with some nouns and adjectives, usually if a noun indicates a proper noun.
> [...][including]
> national and regional adjectives: "an American" (noun), "an American man" (adjective).

from [Wikipedia: Capitalization in English](https://en.wikipedia.org/wiki/Capitalization_in_English#When_to_capitalize)

So I think this should be `with the European Ember community`

**Review by jayjayjpg — COMMENTED** [review]
This looks good and I just left a few comments on typos from my side :+1:

File: index.html (hunk 1 above)
Comment (marcoow) [reply index.html]: fixed 👍

File: index.html (hunk 2 above)
Comment (marcoow) [reply index.html]: fixed 👍

File: index.html (hunk 3 above)
Comment (marcoow) [reply index.html:50]: fixed 👍

**Review by jayjayjpg — APPROVED** [review]

**Pull request merged**